update.sh

```diff
@@ -18,6 +19,19 @@ arch=$(get_arch)
 
 yarnVersion="$(curl -sSL --compressed https://yarnpkg.com/latest-version)"
 
+function in_versions_to_update() {
+  local version=$1
+
+  for version_to_update in "${update_versions[@]}"; do
+    if [[ "$version_to_update" == "$version" ]]; then
```

Comment:
I'll open another issue to address this point, looks like we're mixing bash syntax comparison with pure POSIX syntax, it'll be better to use a single one, especially we're not using any comparison not supported by POSIX syntax but supported by bash, what do you think?

Reply:
#742

Reply:
For those don't need bash enhanced comparison, I prefer to use single one. Though the points here is the mixing of `[ ]` and `[[ ]]`

Reply:
So would it be `[ "$version_to_update" == "$version" ]`?

Reply:
Just a single `=`: `[ "$version_to_update" = "$version" ]`

See also https://serverfault.com/questions/52034/what-is-the-difference-between-double-and-single-square-brackets-in-bash/705144#705144 and http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html. :+1:

Reply:
Thanks!  @tianon for that article! I'm a noob at shell scripting it's nice to see explanations like these!